bridge/standard/cmd/relayer/main.go

```diff
@@ -186,14 +186,17 @@ func main() {
 	signal.Notify(sigc, syscall.SIGINT, syscall.SIGTERM)
 	go func() {
 		<-sigc
+		//nolint:errcheck
```

Comment:
There's enough `//nolint:errcheck` in this PR that we should disable that check imo

Reply:
we don't have configuration for linter: .golangci.yml

Reply:
Can we create one? I don't see the use of adding tens to hundreds of lint exceptions in a PR

Reply:
@aloknerurkar added placeholders